.github/workflows/ci.yml

```diff
@@ -37,5 +37,5 @@ jobs:
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@924441154cf3053034c6513d5e06c69d262fb9a6  # v1.13.0
     with:
       envs: |
-        - linux: withromancal
```

Comment:
Would you restore this? It's required to test opening an association as a `ModelLibrary`:
https://github.com/spacetelescope/roman_datamodels/blob/ab68c3537278140607b7c290fb5fc3cf81a45cd9/tests/test_open.py#L245-L246

Reply:
I just changed the name

Reply:
The `toxini` also now no longer includes `withromancal` (which did not run romancal tests). Here's the run on main:
https://github.com/spacetelescope/roman_datamodels/actions/runs/11978300995/job/33398032820#step:10:398

Where is `open_asn` tested now that there is no `withromancal` tox environment?

Reply:
Oh I see, why don't you just install romancal with the normal tox test.

Reply:
Thanks for the suggestion. I was concerned that would mask potential issues where roman_datamodels could silently depend on something in romancal. Since romancal isn't a dependency most of the tests should pass without romancal (and testing under those conditions validates that). It's only this test that we need romancal for (at the moment) so running it in a separate environment made sense. I'm open to suggestions on other ways to organize this and all in favor of adding a downstream romancal test but just want to make sure we're also testing `open_asn`.

Reply:
Fine, I changed it to an optional